{"name": "test_turn_on_the_wifi_and_try_send_the_message", "status": "failed", "statusDetails": {"message": "AssertionError: assert 'This message... WiFi is ON!!' == 'This message... is Toggled!!'\n  - This message is sent when the WiFi is Toggled!!\n  ?                                       ^^^^^^^\n  + This message is sent when the WiFi is ON!!\n  ?                                       ^^", "trace": "request = <FixtureRequest for <Function test_turn_on_the_wifi_and_try_send_the_message>>, _pytest_bdd_example = {}\n\n    @pytest.mark.usefixtures(*func_args)\n    def scenario_wrapper(request: FixtureRequest, _pytest_bdd_example: dict[str, str]) -> Any:\n        scenario = templated_scenario.render(_pytest_bdd_example)\n>       _execute_scenario(feature, scenario, request)\n\nvenv/lib/python3.10/site-packages/pytest_bdd/scenario.py:187: \n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _\nvenv/lib/python3.10/site-packages/pytest_bdd/scenario.py:157: in _execute_scenario\n    _execute_step_function(request, scenario, step, step_func)\nvenv/lib/python3.10/site-packages/pytest_bdd/scenario.py:127: in _execute_step_function\n    return_value = call_fixture_func(fixturefunc=step_func, request=request, kwargs=kwargs)\nvenv/lib/python3.10/site-packages/_pytest/fixtures.py:890: in call_fixture_func\n    fixture_result = fixturefunc(**kwargs)\n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _\n\n    @then('12 Verify the recently received message on the receiver side')\n    def verify_the_recently_received_message_on_the_receiver_side():\n        \"\"\"12 Verify the recently received message on the receiver side.\"\"\"\n>       assert receiver_messages_screen.fetch_recently_received_message() == \"This message is sent when the WiFi is Toggled!!\"\nE       AssertionError: assert 'This message... WiFi is ON!!' == 'This message... is Toggled!!'\nE         - This message is sent when the WiFi is Toggled!!\nE         ?                                       ^^^^^^^\nE         + This message is sent when the WiFi is ON!!\nE         ?                                       ^^\n\nFeatures/Steps/WiFi_Toggle/test_WiFi_Toggle.py:128: AssertionError"}, "description": "WiFi_Toggle.feature: 03 Turn On the WiFi and try send the message", "start": 1669806612824, "stop": 1669806630678, "uuid": "841f3359-2611-4e09-bd59-76f3732839bb", "historyId": "55cdb3f9503c70216bea03ee785e6076", "testCaseId": "f3217aa666ddaf2e56717a47c11e6f7b", "fullName": "Features.Steps.WiFi_Toggle.test_WiFi_Toggle#test_turn_on_the_wifi_and_try_send_the_message", "labels": [{"name": "tag", "value": "@pytest.mark.usefixtures"}, {"name": "parentSuite", "value": "Features.Steps.WiFi_Toggle"}, {"name": "suite", "value": "test_WiFi_Toggle"}, {"name": "host", "value": "IM-LP-1845.local"}, {"name": "thread", "value": "32718-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "Features.Steps.WiFi_Toggle.test_WiFi_Toggle"}]}